api/importer/package_processor.go

```diff
@@ -101,6 +108,57 @@ func (ipp *ImportPackageProcessor) Process(project string, ip ImportPackage) err
 	return nil
 }
 
+func (ipp *ImportPackageProcessor) validateManifest(
+	manifest *model.ImportManifest, ip ImportPackage) error {
+	re := regexp.MustCompile("^[a-zA-Z0-9_]*$")
+
+	for _, task := range manifest.Tasks {
+		// Check if ID is not empty and formatted properly
+		if task.ID == "" {
+			return fmt.Errorf("task id cannot be empty")
+		} else if !re.MatchString(task.ID) {
+			return fmt.Errorf("task id %s can only consist of alphnumeric characters and underscores", task.ID)
+		}
+
+		// Check if task definition is correct
+		switch task.Type {
```

Comment:
I think `task.APITask` and `task.ResourceTask` shouldn't be allowed to be set in one action.

Reply:
Are there any specific reasons for this? Currently, the `ResourceTask` will not be used when calling an API task and vice versa, but it is not restricted not to provide both.

Reply:
It's not restricted but it's a sign of a badly written manifest

Reply:
Fine, I added a condition that both cannot be set in the same action.